### PR TITLE
Move ostree sysroot scripts to aktualizr repo

### PR DESCRIPTION
**Pull request by lbonn**
It now makes little sense to keep them separate

Sourced from 291cb196330ee28c6e126f9c31778c97cc1df2ae @advancedtelematic/ostreesysroot

### CODE REVIEW

File: src/libaktualizr/opcuabridge/CMakeLists.txt [hunk 1]
```diff
@@ -34,7 +34,7 @@ set(HEADERS boostarch.h
             utility.h
             versionreport.h)
 
-set_source_files_properties(${SOURCES} ${PROJECT_SOURCE_DIR}/third_party/open62541/open62541.c PROPERTIES COMPILE_FLAGS "-Wno-unused-parameter -Wno-float-equal -Wno-logical-op -Wno-unknown-warning-option -Wno-unused-function -Wno-switch-default")
+        set_source_files_properties(${SOURCES} ${PROJECT_SOURCE_DIR}/third_party/open62541/open62541.c PROPERTIES COMPILE_FLAGS "-Wno-unused-parameter -Wno-float-equal -Wno-logical-op -Wno-unknown-warning-option -Wno-unused-function -Wno-switch-default -Wno-cast-function-type")
```
Comment (patrickvacek) [inline src/libaktualizr/opcuabridge/CMakeLists.txt]: The indent here seems unnecessary.

**Comment by lbonn** [comment]
Here is the history. I did not use the merge trick but just cherry-picked everything.

**Comment by patrickvacek** [comment]
Nice! That works, too. I actually thought the history was longer, so that's not so bad.

**Comment by lbonn** [comment]
@patrickvacek no objections?

**Pull request merged**